docs/onboarding.md

```diff
@@ -0,0 +1,157 @@
+[Table of contents](./README.md#table-of-contents)
+
+Onboarding
+==========
+
+This document explains the architecture and process allowing a cozy instance owner to register to its cozy instance.
+
+Compatibility with the current developments on cozy onboarding is a goal : [The following documents has been consulted for this proposal]( https://github.com/cozy/cozy-proxy/blob/bf9af7f2342e3fc183a8b4e72bcedb909afa3eb8/docs/client/)
+
+## Instance creation
+
+Creating an instance is done through CLI or through the (future) partner farm manager system. Some **settings** can be pre-defined on instance creation. ([doc](./instance.md#creation)).
+
+The CLI also allows to specify which source to use for `onboarding` and `home` applications. The defaults will be hosted on `github.com/cozy`.
+
+After creation, an instance has a `registerToken` generated randomly.
```

Comment:
So, if I understand correctly, it implements option 2) with a unix timestamp in the message so that it can expire without storing an additional date in DB. I think we would use a JWT for such things if your question is more on the formalism part.

I still don't think we don't want to introduce a global private key in this situation, a random token per user should hopefully be sufficient.